BusBoard.Web/Views/Home/BusInfo.cshtml

```diff
@@ -1,10 +1,61 @@
-﻿@model BusBoard.Web.ViewModels.BusInfo
+﻿@using Microsoft.AspNetCore.Html
+@using BusBoard.Api.Tfl
+@using BusBoard.Api.Tfl.Responses
+@using BusBoard.Web.ViewModels
+@model BusBoard.Web.ViewModels.BusInfo
 @{
-  ViewBag.Title = "Bus Information";
+    ViewBag.Title = "Bus Information";
+}
+
+@helper RenderStop(string stopName, IEnumerable<Prediction> predictions)
```

Comment:
These two helper functions as well I would inline. I wouldn't generally expect to find any functions in cshtml files 🙂

Reply:
Changed in 113e677 🙂 I was worried about the amount of nesting that this results in (doubly nested `for` loops inside a conditional) and would normally look to break that up in normal C#. In cshtml is inlining and nesting preferred over separating out into functions then?

Reply:
So I wouldn't use functions in cshtml files as they should have as little logic in them as possible. If something is too long or too nested then I would use partials. In this case I think it's borderline. You could have a stop and/or prediction partial, but the html for each is quite simple so I wouldn't mind if you keep it as it is either. Does that make sense? 🙂